package.json

```diff
@@ -36,6 +36,7 @@
     "@scure/bip39": "^1.1.1",
     "@twogate/ngx-photo-gallery": "^1.4.0",
     "@types/sharedworker": "^0.0.91",
+    "angularx-qrcode": "^15.0.1",
```

Comment:
There is already another QRCode library in use, but we'll merge this in and refactor later.